### PR TITLE
feat: add functions to yaml

### PR DESCRIPTION
**Pull request by rui-mo**
This PR added starts_with, ends_with, contains functions to yaml files.

### CODE REVIEW

File: extensions/functions_comparison.yaml [hunk 1]
```diff
@@ -73,4 +73,12 @@ scalar_functions:
           - value: any1
         return: BOOLEAN
         nullability: DECLARED_OUTPUT
-
+  -
+    name: "in"
+    description: Whether a value is in a list of values.
+    impls:
+      - args:
+          - value: any1
```
Comment (jvanstraten) [inline extensions/functions_comparison.yaml:81]: I think you meant this?

```suggestion
          - value: T
```

**Review by jvanstraten — CHANGES_REQUESTED** [review]

**Comment by jacques-n** [comment]
I think you're looking for the orlist expression (as opposed to introducing in).

For the character matching functions, I suggest you also include all permutations of fixed character, string and varchar combinations.

File: extensions/functions_comparison.yaml (hunk 1 above)
Comment (rui-mo) [reply extensions/functions_comparison.yaml]: The function we need is already covered by OrList expression. Removed this.

**Comment by rui-mo** [comment]
> I think you're looking for the orlist expression (as opposed to introducing in).
> 
> For the character matching functions, I suggest you also include all permutations of fixed character, string and varchar combinations.

Thanks for your comment. Removed in function and added the permutations for starts_with, ends_with, contains.

**Comment by jacques-n** [comment]
Please also update commit title. Should be feat and more specific

**Comment by rui-mo** [comment]
> Please also update commit title. Should be feat and more specific

Updated. Thank you.

**Review by jacques-n — APPROVED** [review]
Looks mostly good. I will update the title to be more explicit when I merge.

**Pull request merged**